source/fundamentals/connection/connect.txt

```diff
@@ -0,0 +1,123 @@
+.. _csharp-connect-to-mongodb:
+
+================
+Connection Guide
+================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide shows you how to connect to a MongoDB instance or replica set
+deployment using the {+driver-short+}.
+
+Connection URI
+--------------
+
+A **connection URI**, also known as a *connection string*, tells the driver how to connect to a MongoDB deployment and how to behave while connected.
+
+A standard connection string includes the following pieces:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Piece
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password@``
+
+     - Optional. Authentication credentials. If you include these, the client will authenticate the user against the database specified in ``authSource``. 
+
+   * - ``host[:port]``
+
+     - Required. The host and optional port number where MongoDB is running. If you don't include the port number, the driver will use the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this piece, the client will authenticate the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. See
+       :ref:`csharp-connection-options` for a full description of
+       these options.
+
+To use a connection URI, pass it as a string to the ``MongoClient`` constructor. In the 
+following example, the driver uses a sample connection URI to connect to a MongoDB 
+instance on port ``27017`` of ``localhost``: 
+
+.. literalinclude:: /includes/fundamentals/code-examples/connection/LocalConnection.cs
+   :language: csharp
+
+.. tip::
+
+    See :manual:`the MongoDB Manual</reference/connection-string>`: for more information about creating a connection string.
+
+MongoClientSettings
+-------------------
+
+You can use a ``MongoClientSettings`` object to configure the connection in code
+rather than in a connection URI. To use a ``MongoClientSettings`` object, create an 
+instance of the class and pass it as an argument to the ``MongoClient`` constructor. 
+
+In the following example, the driver uses a ``MongoClientSettings`` object to connect to a 
+MongoDB instance on port ``27017`` of ``localhost``:
+
+.. literalinclude:: /includes/fundamentals/code-examples/connection/MongoClientSettings.cs
+   :language: csharp
+   :dedent:
+
+Other Connection Targets
+------------------------
+
+Connect to a Standalone Deployment
```

Comment:
If by `standalone deployment` you mean a single mongod instance, then it's not really true. The same connection string will work for sharded cluster too

Reply:
I'm ok to remove it